dockstore-webservice/pom.xml

```diff
@@ -619,9 +619,219 @@
             <groupId>com.github.jsonld-java</groupId>
             <artifactId>jsonld-java</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.elasticsearch</groupId>
+            <artifactId>elasticsearch</artifactId>
+            <exclusions>
+                <!-- Excluding lucene saves 2 MB, everything else combined saves 1 MB -->
+<!--                <exclusion>-->
+<!--                    <groupId>com.carrotsearch</groupId>-->
+<!--                    <artifactId>hppc</artifactId>-->
+<!--                </exclusion>-->
+                <exclusion>
+                    <groupId>com.tdunning</groupId>
+                    <artifactId>t-digest</artifactId>
+                </exclusion>
+                <exclusion>
```

Comment:
Don't have to hurry to change this, but this exclusion probably doesn't do anything since joda-time is included via other dependencies.

Reply:
systematically excluding every transitive dependency that doesn't throw an error during builds/tests in order to ensure it's the smallest size possible.

Reply:
Oh, if you want to exclude everything, there might be an easier way
Maybe give a wildcard https://issues.apache.org/jira/browse/MNG-3832 a try